client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -96,6 +88,9 @@ const ClinicalDetails: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Element
     const saveForm = (e: any) => {
         e.preventDefault();
         const values = getValues();
+        if(values.isSymptomsStartDateUnknown) {
+            values.symptomsStartDate = null;
+        }
         saveClinicalDetails(values as ClinicalDetailsData, epidemiologyNumber, investigatedPatientId);
```

Comment:
Will sending the values as clinicalDetailsData omit the `isSymptomsStartDateUnknown` from the object?

Reply:
No it won't, I added isSymptomsStartDateUnknown to ClinicalDetailsData